### PR TITLE
User Journey

### PR DESCRIPTION
**Pull request by MichaelChinn**
new pr ensures large content files not added to history

steps:
go to /landing
click [TUTORIAL] in header
click [NEXT] to go through user journey
on last step click [FINISH] and return to landing

### CODE REVIEW

File: jazz-ui/src/app/pages/user-journey/user-journey.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import {environment} from "../../../environments/environment";
```
[the remark this answers is not in the document]
Comment (hks93) [reply jazz-ui/src/app/pages/user-journey/user-journey.ts]: Please Test - the right file is getting imported all the time

File: jazz-ui/src/app/pages/user-journey/user-journey.component.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import {Component, OnInit, ViewEncapsulation} from '@angular/core';
+import { Router } from '@angular/router';
+import {UserJourney} from "./user-journey";
+
+@Component({
+  selector: 'user-journey',
+  templateUrl: './user-journey.component.html',
+  styleUrls: ['./user-journey.component.scss'],
+  encapsulation: ViewEncapsulation.None
```
[the remark this answers is not in the document]
Comment (MichaelChinn) [reply jazz-ui/src/app/pages/user-journey/user-journey.component.ts]: Dynamic styles moved to global namespace

File: jazz-ui/src/app/pages/user-journey/user-journey.component.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  setStep(index) {
+    let previousIndex = this.stepIndex;
+    this.stepIndex = index;
+    this.currentStep = this.steps[this.stepIndex];
+    this.last = (index === this.steps.length - 1);
+    let playingVideo: any = document.getElementById('user-journey-video-' + this.stepIndex);
+    if(playingVideo) {
+      playingVideo.currentTime = 0;
+      setTimeout(() => {
+        if(this.stepIndex === index) playingVideo.play();
```
[the remark this answers is not in the document]
Comment (MichaelChinn) [reply jazz-ui/src/app/pages/user-journey/user-journey.component.ts]: used to determine current video during asynchronous callback.

File: .gitignore [hunk 4]
```diff
@@ -12,4 +12,5 @@ node_modules
 */target/
 .nyc_output/
 npm-debug.log
-*/package-lock.json
\ No newline at end of file
+*/package-lock.json
+/.idea
```
[the remark this answers is not in the document]
Comment (MichaelChinn) [reply .gitignore]: removed

File: jazz-ui/src/app/pages/user-journey/user-journey.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MichaelChinn) [reply jazz-ui/src/app/pages/user-journey/user-journey.ts]: Tested, environment file imported based on ng build environment flag

**Pull request merged**